groups-react/README.md

```diff
@@ -0,0 +1,12 @@
+# io.Connect Desktop Groups in React
+
+This is a template for creating a React Groups application for [**io.Connect Desktop**](https://interop.io//).
+
+## Prerequisites
+
+For a **io.Connect Desktop** project, you need to have [**io.Connect Desktop**](https://interop.io//) 9.0.0 or newer.
+
+## Usage
+
+- Run `npm install` to install all dependencies.
+- Run `npm run start` to start the app.
```

Comment:
\- Copy the \`webGroup.json\` configuration file and replace the existing Web Group App definition in the \`%LocalAppData%\interop.io\io.Connect Desktop\Desktop\config\apps\` folder.
\- Set the \`"groupType"\` property of the \`stickywindows.json\` file of \*\*io.Connect Desktop** to \`"Web"\`.
\- Start \*\*io.Connect Desktop** to start using and modifying your Web Group App.